counterparty-core/counterpartycore/lib/messages/order.py

```diff
@@ -762,6 +766,9 @@ def match(db, tx, block_index=None):
 
             if block_index >= 313900 or config.TESTNET or config.REGTEST:  # Protocol change.
                 min_btc_quantity = 0.001 * config.UNIT  # 0.001 BTC
+                if util.enabled("fix_min_btc_quantity", block_index):
+                    # we subtract 1 because the <= instead of < like when checking orders
```

Comment:
Maybe we make a function `new_min_btc_quantity` as this computation is performed in multiple places?

Reply:
hum.. no if fact it's not exactly the same in the two places (there is a -1 here)